content/wireless.md

```diff
@@ -206,4 +348,8 @@ Sometimes the newest version of the firmware will clear up occasional bugs.  Ple
 
 ### Windows Dual Boot
 
-If you are dual booting Windows, you may lose access to your wifi card entirely after running driver/OS updates in Windows. You may be able to gain access to your wifi card again by disabling "Fast Startup" in the Windows power options before booting back into Pop!_OS.
+If you're dual booting Windows, you may lose access to your Wi-Fi card entirely after running driver/OS updates in Windows. You may be able to gain access to your Wi-Fi card again by disabling "Fast Startup" in the Windows power options before booting back into Pop!_OS.
+
+## Contact System76 Support
+
+If you purchased a System76 computer and you’ve tried all the steps above, but your wireless connection still isn’t working as expected, please collect the output from the diagnostic commands and [contact support](https://system76.com/contact/support).
```

Comment:
You didn't need to remove System76 from the link name. It should say `contact System76 support`.